### DISCUSSION TITLE
Search Results Webparts Automatically Refreshing (Unwanted)

### DISCUSSION THREAD

**Discussion by dontthroworanges** [microsoft-search/pnp-modern-search#4062] [Q&A] [answered]
I am using the URL Fragment method to get a query into the Search Box webpart from another page. The fragment successfully gets into the search box and the search results webpart accurately shows the correct results. However, after a few seconds the results web part refreshes to its default state and the proper results are lost. I do have a few different results webparts on the page that are tied to a verticals webpart. Maybe something in there is causing the issue? Anything I should be looking out for? Any help would be appreciated. Thanks!

![image](https://github.com/user-attachments/assets/53944a25-27f0-4a7d-9cf8-a6f5004618c1)

**Comment by wobba** [comment]
What happens if you use a query parameter instead of a fragment as a workaround?

**Reply by dontthroworanges** [reply, marked as answer]
Sorry for the late reply as it took me a while to get back around to this issue. I tried the Query Parameter method you suggested but I realized that this method wasn't ideal since the target results web-part is too generic and I didn't want to limit its scope. 

Regardless, I actually found a solution. I inserted the flag: ```data-interception="off"``` into my handlebars for the column and that completely solved the issue. Again, maybe this isn't the best method, but works well enough to report back in case anyone else has the issue.
